src/platform/emotes/emote.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  generateValidator,
+  JSONSchema,
+  ValidateFunction
+} from '../../validation'
+import { Rarity } from '../../dapps/rarity'
+import { I18N } from '../common'
+import { EmoteRepresentation } from './representation'
+import { EmoteData } from './data'
+
+/** @alpha */
+export type Emote = {
```

Comment:
Is emote an entity of the catalyst?

Reply:
I think so, how shall that be specified?  I dont see it in the wearables: https://github.com/decentraland/common-schemas/blob/10a57e35c94940f6c6a66ef45d102e2099be946a/src/platform/wearables/wearable.ts#L17